mne_bids/mne_bids.py

```diff
@@ -619,6 +644,33 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                                      output_path=output_path, make_dir=False,
                                      overwrite=False, verbose=verbose)
 
+    processing = None
+    proc_history = raw.info['proc_history']
+    if proc_history != []:
+        sss = False
+        tsss = False
+        for ph in proc_history:
+            max_info = ph.get('max_info')
+            if max_info:
+                if (max_info['sss_info'] != dict() or
+                        max_info['sss_ctc'] != dict() or
+                        max_info['sss_cal'] != dict()):
+                    sss = True
+                if max_info['max_st'] != dict():
```

Comment:
`elif` ...

Reply:
we actually want just an `if` statement here.
There can be `sss` processing data and `tsss` processing data, so something having `sss` data doesn't mean it cannot have `tsss` data (and I think in fact it *has* to have the `sss` data for the `tsss processing)
(Althought I really don't know much about this processing or how it works, just going off what I have noticed in the files)

Reply:
I'm not exactly sure what you mean. SSS and tSSS are different methods. It cannot be both if I'm not mistaken. Basically, tSSS is an extension of SSS ...

Reply:
yes, but the processing information for sss data is there for tsss data also. Or have I misunderstood the data I have looked at?